jdaviz/configs/cubeviz/plugins/tools.py

```diff
@@ -2,28 +2,48 @@
 import os
 
 from glue.config import viewer_tool
+from glue_jupyter.bqplot.image import BqplotImageView
 from glue.viewers.common.tool import CheckableTool
 
 from jdaviz.configs.imviz.plugins.tools import _MatchedZoomMixin
 from jdaviz.core.events import SliceToolStateMessage
-from jdaviz.core.tools import PanZoom, SinglePixelRegion
+from jdaviz.core.tools import PanZoom, BoxZoom, SinglePixelRegion
 
 __all__ = []
 
 ICON_DIR = os.path.join(os.path.dirname(__file__), '..', '..', '..', 'data', 'icons')
 
 
+class _CubevizMatchedZoomMixin(_MatchedZoomMixin):
+    match_keys = ('x_min', 'x_max', 'y_min', 'y_max')
+    disable_matched_zoom_in_other_viewer = False
+
+    def _is_matched_viewer(self, viewer):
+        return isinstance(viewer, BqplotImageView)
+
+
 @viewer_tool
-class SimpleMatchPanZoom(_MatchedZoomMixin, PanZoom):
```

Comment:
Is there a reason to rename this? It is not being used anywhere else.

Reply:
Because I thought this gave a little more context to what it meant and hopefully avoid any misuse or bugs later (if we ever want multiple cube support in the future, etc).  But I don't feel too strongly about the names if we reach a consensus on something else (or going back to "simple")

Reply:
"NoWCS" could be confusing too. Maybe someone will think to avoid this tool because their data have WCS without understanding that you can still pixel link despite that. 🤷 

Let's see what others say.

Reply:
I agree with @pllim. The docstring makes it clear, and the all lowercase `panzoommatchnowcs` is hard for my brain to parse.

Reply:
I am neutral on that. What about you, Brett?